tests_integration/test_data/projects/snowpark_with_package/requirements.txt

```
@@ -0,0 +1,3 @@
+snowflake-snowpark-python
+syrupy
```

```suggestion
syrupy. # This should be a package that is not in snowflake anaconda
```
Just a note to our future selves